code/scripting/lua.cpp

```diff
@@ -363,11 +363,15 @@ void script_state::OutputLuaMeta(FILE *fp)
 	//***Enumerations
 	fprintf(fp, "<dt id=\"Enumerations\"><h2>Enumerations</h2></dt>");
 	for (uint32_t i = 0; i < Num_enumerations; i++) {
-		//WMC - This is in case we ever want to add descriptions to enums.
-		//fprintf(fp, "<dd><dl><dt><b>%s</b></dt><dd>%s</dd></dl></dd>", Enumerations[i].name, Enumerations[i].desc);
-
-		//WMC - Otherwise, just use this.
-		fprintf(fp, "<dd><b>%s</b></dd>", Enumerations[i].name);
+		// WMC - This is in case we ever want to add descriptions to enums.
```

Comment:
WMC's descriptions comment should appear immediately prior to his other comment, because that is what "otherwise" refers to.

Be careful when you make code changes that you don't introduce errors in comments.  They can't be caught in the same way as errors in code.

Reply:
Can we change his second comment to 		
`// WMC - For now, print to the file without the description.`

Reply:
The two comments are two sides of the same coin.  They are conceptually related and should be linked.  They *were* linked until you put the code in the middle and separated them.

Just move your code to the top of the `for` loop.